### PR TITLE
Add REWARD_Ontology

### PR DESCRIPTION
**Pull request by hrysakis**
Please proceed with the pull request of adding the REWARD_Ontology folder.

### CODE REVIEW

**Comment by dgarijo** [comment]
@hrysakis, you are deleting the main readme in this PR. 
Please only add the files you want to add in your PR.

File: REWARD_Ontology/.htaccess [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Header set Access-Control-Allow-Origin *
+Header set Access-Control-Allow-Headers DNT,X-Mx-ReqToken,Keep-Alive,User-Agent,X-Requested-With,If-Modified$
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^$ https://www.cap-a.eu/model/reward.ttl[[R=302,L]
```
Comment (davidlehn) [inline REWARD_Ontology/.htaccess]: typo

**Comment by davidlehn** [comment]
Also can't remove main .htaccess.

File: REWARD_Ontology/.htaccess (hunk 1 above)
Comment (hrysakis) [reply REWARD_Ontology/.htaccess]: this typo has been fixed.

**Comment by hrysakis** [comment]
Thanks for your help @davidlehn  and @dgarijo and I'm sorry for any inconvenience. Is it OK now?

File: REWARD_Ontology/.htaccess [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Header set Access-Control-Allow-Origin *
+Header set Access-Control-Allow-Headers DNT,X-Mx-ReqToken,Keep-Alive,User-Agent,X-Requested-With,If-Modified$
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^$ https://www.cap-a.eu/model/reward.ttl/$1 [R=302,L]
```
Comment (davidlehn) [inline REWARD_Ontology/.htaccess]: You don't have any pattern that would be used in that `$1` substitution.  Also looks non-standard to add something after a ttl file name like that.  What are you trying to do there?  I'd recommend testing rules locally if you can.

**Comment by davidlehn** [comment]
@hrysakis No.  You are still removing the root README.md and .htaccess.

File: REWARD_Ontology/.htaccess (hunk 2 above)
Comment (hrysakis) [reply REWARD_Ontology/.htaccess]: My intention is to forward the url: https://w3id.org/REWARD_Ontology to https://www.cap-a.eu/model/reward.ttl.

Comment (hrysakis) [reply REWARD_Ontology/.htaccess]: According to the last commit I think that now it should be OK. thanks.

**Comment by hrysakis** [comment]
> 
> 
> @hrysakis No. You are still removing the root README.md and .htaccess.

I have created the README,md file and htacess file in my repository. Can you help me further what other actions are needed because I'm not sure that I understandregarding the removing you mentioned above.
Thanks.

**Comment by dgarijo** [comment]
@hrysakis, in your PR you are deleting the main readme and htaccess of w3id, look at the "Files Changed" for more information. 
You can revert some of your commits, or you can close the PR, update the latest version of the repository and then open a new PR.

**Pull request closed without merging**